Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -23,6 +23,11 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import Metal
+
+private let WGPU_COPY_STRIDE_UNDEFINED = 0xffffffff
```

Comment:
Can we import this value from WebGPU.h?

If there's something stopping us from using a #define directly, can we change the #define to a static const, or add a using declaration for Swift like we do for templates?